applications/ismp.md

```diff
@@ -0,0 +1,184 @@
+
+1. Seun Lanlege - Founder, Mad Scientist at Polytope Labs and [Polkadot Fellowship Member](https://github.com/polkadot-fellows/seeding/pull/33). Previously core developer at Parity Tech, Worked on Ethereum and Polkadot with over 4 years of industry experience.
+
+2. David Salami - Scientist at Polytope Labs and [Polkadot Fellowship Member](https://github.com/polkadot-fellows/seeding/pull/38). Previously Senior Blockchain Engineer at [Composable Finance](https://github.com/ComposableFi/) and [Webb](https://github.com/webb-tools).
+
+3. Damilare Akinlose - Lab Intern at Polytope Labs and [Polkadot Fellowship Member](https://github.com/polkadot-fellows/seeding/pull/41). Previously Blockchain Engineer at [Webb](https://github.com/webb-tools)
+
+4. Femi Bankole - Blockchain engineer at [Matchx_iot](https://matchx.io/) + [MXC Foundation](https://www.mxc.org/) and Lab Intern at Polytope Labs.
+
+5. Jesse Chejieh - [Polkadot Fellowship Member](https://github.com/polkadot-fellows/seeding/pull/25).
+
+### Research Publications
+
+ - [Consensus Proofs](https://research.polytope.technology/consensus-proofs)
+ - [State (Machine) Proofs](https://research.polytope.technology/state-(machine)-proofs)
+ - [Merkle Mountain Range Proofs](https://research.polytope.technology/merkle-mountain-range-multi-proofs)
+ - [Merkle Multi Proofs](https://research.polytope.technology/merkle-multi-proofs)
+
+### Team Code Repos
+- [sync-committee-rs](https://github.com/polytope-labs/sync-committee-rs)
+- [Merkle Patricia Trie](https://github.com/polytope-labs/patricia-merkle-trie)
+- [Solidity Merkle Trees](https://github.com/polytope-labs/solidity-merkle-trees)
+- [Simnode](https://github.com/polytope-labs/sc-simnode)
+
+
+### Team GitHub Profiles
+
+- [Seun Lanlege](https://github.com/seunlanlege/)
+- [David Salami](https://github.com/Wizdave97/)
+- [Damilare Akinlose](https://github.com/dharjeezy/)
+- [Femi Bankole](https://github.com/iTranscend/)
+- [Jesse Chejieh](https://github.com/Doordashcon/)
+
+### Team LinkedIn Profiles
+
+- [Seun Lanlege](https://www.linkedin.com/in/seunlanlege/)
+- [David Salami](https://www.linkedin.com/in/david-salami-188aa8170)
+- [Damilare Akinlose](https://www.linkedin.com/in/damilare-akinlose-6289b5b8)
+- [Femi Bankole](https://www.linkedin.com/in/femibankole)
+- [Jesse Chejieh](https://www.linkedin.com/in/jesse-chejieh-47020322a)
+
+
+## Development Status :open_book:
+State-proof based parachain messaging has been discussed on the [Polkadot Forum](https://forum.polkadot.network/t/state-proof-based-parachain-parachain-messaging/2214/4).
+
+And some Updates to cumulus, required for parachains to read the relay chain state has been [approved](https://github.com/paritytech/cumulus/pull/2268)
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+- **Total Estimated Duration:** 6 Months
+- **Full-Time Equivalent (FTE):** 6
```

Comment:
The totals don't match the sum of the milestones.

Reply:
Hello @semuelle, This was an oversight, we previously had three milestones, the last milestone included configuration to cumulus which we has been implemented and merged.

The estimated duration has been updated, pls review.